runtime/doc/starting.txt

```diff
@@ -381,76 +350,6 @@ argument.
 							*-W*
 -W {scriptout}	Like -w, but do not append, overwrite an existing file.
 
---remote [+{cmd}] {file} ...
-		Open the {file} in another Vim that functions as a server.
-		Any non-file arguments must come before this.
-		See |--remote|.
```

Comment:
I wanted to keep these around as a living record of which clientserver features we may need to re-implement or provide ways to mimic. Having to go back and cross-reference legacy Vim documentation is more difficult.